server/container/guice/cassandra-rabbitmq-guice/src/main/java/org/apache/james/modules/blobstore/BlobStoreChoosingModule.java

```diff
@@ -43,11 +49,47 @@
 import com.google.common.annotations.VisibleForTesting;
 import com.google.inject.AbstractModule;
 import com.google.inject.Provides;
+import com.google.inject.Scopes;
 import com.google.inject.multibindings.Multibinder;
+import com.google.inject.name.Names;
 
 public class BlobStoreChoosingModule extends AbstractModule {
     private static final Logger LOGGER = LoggerFactory.getLogger(BlobStoreChoosingModule.class);
 
+    public static class CacheDisabledModule extends AbstractModule  {
+        @Provides
+        @Named(MetricableBlobStore.BLOB_STORE_IMPLEMENTATION)
+        @Singleton
+        BlobStore provideBlobStore(@Named(CachedBlobStore.BACKEND) BlobStore blobStore) {
```

Comment:
Question, as I'm not sure I understand, why declaring `BACKEND` constant in `CachedBlobStore`, if we use it when we don't have the cache?

Reply:
That's just an injection key (by name)

The disable one simply says "what we use as a backend for our cache" should be the blobStore we use.

So without the cache.